src/main/java/response/FileExtension.java

```diff
@@ -0,0 +1,59 @@
+package response;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public enum FileExtension {
+
+    CSS(".css", "text/css", Constants.STATIC),
+    PNG(".png", "image/png", Constants.STATIC),
+    JS(".js", "text/js", Constants.STATIC),
+    EOT(".eot", "application/vnd.ms-fontobject", Constants.STATIC),
+    SVG(".svg", "image/svg+xml", Constants.STATIC),
+    TTF(".ttf", "application/octet-stream", Constants.STATIC),
+    WOFF(".woff", "application/font-woff", Constants.STATIC),
+    WOFF2(".woff2", "application/font-woff", Constants.STATIC),
+    ICO(".ico", "image/x-icon", Constants.TEMPLATE),
+    HTML(".html", "text/html", Constants.TEMPLATE);
+
+    private static Map<String, FileExtension> fileExtensions = new HashMap<>();
+
+    private String extension;
+    private String contentType;
+    private String filePath;
+
+    private static class Constants {
+        public static final String STATIC = "./static";
+        public static final String TEMPLATE = "./templates";
+    }
+
+    static {
+        for(FileExtension fileExtension : FileExtension.values()){
+            fileExtensions.put(fileExtension.extension, fileExtension);
+        }
+    }
+
+    FileExtension(String extension, String contentType, String filePath) {
+        this.extension = extension;
+        this.contentType = contentType;
+        this.filePath = filePath;
+    }
+
+    public String getExtension() {
+        return extension;
+    }
+
+    public String getContentType() {
+        return contentType;
+    }
+
+    public String getFilePath() {
+        return filePath;
+    }
+
+    public static FileExtension getFileExtensionToExtension(String extension){
+        return Optional.of(fileExtensions.get(extension)).orElse(HTML);
```

Comment:
`Optional.of`는 not-null로만 취급 되기 때문에 `fileExtensions.get`이 null을 반환할 때,
`NullPointerException`이 발생해서 `orElse` 부분이 무시됩니다.
```suggestion
        return Optional.ofNullable(fileExtensions.get(extension)).orElse(HTML);
```
이 더 맞는 용법일것 같아요.
하지만 이 경우에는 Optional 없이도 간단히 이용할 수 있겠네요
```suggestion
        return fileExtensions.getOrDefault(extension, HTML);
```

Reply:
넵 알겠습니다 해당 부분 반영하겠습니다!